package.json

```diff
@@ -12,6 +12,7 @@
     "build:pficons": "gulp pfIcons",
     "build:pficonfont": "gulp pfIconFont",
     "clean": "gulp clean",
+    "superclean": "npm install && npm run clean && git clean -dfx",
```

Comment:
I don't think you need to run an install first? unless there is a goal in doing so that i'm unaware of? But the git clean is going to immediately remove what you just installed.

Reply:
I'm not sure why, but if there are significant package.json files, often if you don't run `npm install` first, you'll get a bunch of errors running `yarn clean` and/or `git clean -dfx`. @srambach ran into it today when pulling in the latest dependency updates - running `yarn install` before `yarn clean && `git clean -dfx` addressed it.